influxql/ast.go

```diff
@@ -2019,6 +2019,23 @@ func (s *CreateContinuousQueryStatement) RequiredPrivileges() ExecutionPrivilege
 	return ep
 }
 
+func (s *CreateContinuousQueryStatement) validate() error {
+	interval, err := s.Source.GroupByInterval()
+	if err != nil {
+		return err
+	}
+
+	if s.ResampleFor != 0 {
+		if s.ResampleEvery != 0 && s.ResampleEvery > interval {
+			interval = s.ResampleEvery
+		}
+		if interval > s.ResampleFor {
+			return fmt.Errorf("resample duration too low: must be a minimum of %s, got %s", FormatDuration(interval), FormatDuration(s.ResampleFor))
```

Comment:
Would it be more clear to the user if the error message explained the rule instead of values for the specific query?
```
FOR duration must be >= GROUP BY time duration
```